src/app/workshop-details/workshop-details.component.html

```diff
@@ -81,8 +88,8 @@ <h2 class="highlight-headings">Workshop Link(s)</h2>
                 </div>
             </div>
 
-            <a class="prev" (click)="plusSlides(-1)">&#10094;</a>
-            <a class="next" (click)="plusSlides(1)">&#10095;</a>
+            <a title="Go to previous image" class="prev" (click)="plusSlides(-1)">&#10094;</a>
```

Comment:
go to previous photography workshop image?

Reply:
these are not used for SEO through text.. just used for proving usability of website